infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -115,6 +103,18 @@ public LayoutServer(@Nonnull ServerContext serverContext) {
         }
     }
 
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651438)